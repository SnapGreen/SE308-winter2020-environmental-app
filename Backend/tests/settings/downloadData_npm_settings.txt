settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	FILENAME: fakefile
	FULLPATH: fakeurl/fakefile
	LOGDIR: Backend/logs/
	CURRENTLATEST: FoodData_Central_branded_food_csv_2020-04-29.zip,202005010423,74439754
	DONE_UPLOADING: false
	DOWNLOADLOGDIR: Backend/logs/downloads/
	USAGE:
		Usage: ./downloadData.sh <filename> <url> [OPTION] (-h for help)
	HELP:
		Usage: ./downloadData.sh <filename> <url> [OPTION] (-h for help)
			-b: bypass debug (will skip settings check)
			-s: output settings only
			-h: print help
