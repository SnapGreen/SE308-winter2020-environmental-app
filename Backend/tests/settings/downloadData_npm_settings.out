settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	FILENAME: fakefile
	FULLPATH: fakeurl/fakefile
	LOGDIR: 
	CURRENTLATEST: 
	DONE_UPLOADING: 
	DOWNLOADLOGDIR: downloads/
	USAGE:
		Usage: ./downloadData.sh <filename> <url> [OPTION] (-h for help)
	HELP:
		Usage: ./downloadData.sh <filename> <url> [OPTION] (-h for help)
			-b: bypass debug (will skip settings check)
			-n: test with settings relative to repo root
			-ns: output settings relative to repo root only
			-s: output settings only
			-t: test mode (silent)
			-h: print help
